### PR TITLE
update load image tutorial

### PR DESCRIPTION
**Pull request by wyli**
follow-up of https://github.com/Project-MONAI/MONAI/pull/6008, adding more examples

### Checks

- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [x] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [ ] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

File: modules/load_medical_images.ipynb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (KumoLiu) [inline modules/load_medical_images.ipynb:1]: <p>Maybe should updated to be "Download an example image"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (KumoLiu) [inline modules/load_medical_images.ipynb:1]: <p>Seems doc-string didn't update here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (KumoLiu) [inline modules/load_medical_images.ipynb:1]: <p>Maybe add a markdown cell up this cell.</p><p>"<code>SaveImage</code> with different backends "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

**Review by KumoLiu — APPROVED** [review]
Thanks for the update, looks good to me.

**Comment by mingxin-zheng** [comment]
Hi @wyli , I am curious if the `# noqa: N813` in `import SimpleITK as sitk  # noqa: N813` affects any of cicd pipelines in the tutorial repo?

**Pull request merged**